<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListWidget" name="lstSection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="AreaSettings">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="areaGeneral"/>
       <widget class="QWidget" name="areaVoIP">
        <widget class="IPCtrl" name="txtHostVoIP">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>109</width>
           <height>22</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtPortVoIP">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>70</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>5038</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>50</y>
           <width>52</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Host :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>50</y>
           <width>52</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Port :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtUserNameVoIP">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>130</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtPassVoIP">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>130</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
        <widget class="QListWidget" name="lstServer">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>121</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>200</y>
           <width>311</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cmdDelServer">
            <property name="text">
             <string>Delete Server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdAddServer">
            <property name="text">
             <string>Add Server</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>110</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>User :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>110</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Password :</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="areaCdr">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="grpRecord">
           <property name="title">
            <string>Call Detail Record</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <widget class="QComboBox" name="cmbDataBase">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>161</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>MariaDB(MySQL)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PostgreSQL</string>
             </property>
            </item>
           </widget>
           <widget class="QTextEdit" name="txtSQLQuery">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>260</y>
              <width>451</width>
              <height>61</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>161</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Type CDR</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="txtdbName">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>140</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="txtTableName">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>180</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="txtUserName">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>140</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="txtPassword">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>180</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>240</y>
              <width>161</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>SQL Query:</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>181</width>
              <height>42</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Host:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Port:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="IPCtrl" name="txtHostCDR">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>5</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>5</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="txtPort">
               <property name="minimumSize">
                <size>
                 <width>55</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>55</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>3306</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cmdSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IPCtrl</class>
   <extends>QFrame</extends>
   <header>IPFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
